docs/python.rst

```diff
@@ -41,4 +41,26 @@ You can then use the loaded interpreter to parse text:
 
     interpreter.parse(u"The text I want to understand")
 
-which returns the same ``dict`` as the HTTP api would (without emulation).
\ No newline at end of file
+which returns the same ``dict`` as the HTTP api would (without emulation).
+
+If multiple models are created, it is reasonable to share components between the different models. E.g.
+the ``'init_spacy'`` component, which is used by every pipeline that wants to have access to the spacy word vectors,
+can be cached to avoid storing the large word vectors more than once in main memory. To use the caching,
+a ``ComponentBuilder`` should be passed when loading and training models, e.g.:
+
+.. testcode::
+
+    from rasa_nlu.model import Metadata, Interpreter
+    from rasa_nlu.components import ComponentBuilder
+    config = RasaNLUConfig("config_spacy.json")
+
+    # For simplicity we will load the same model twice, usually you would want to use the metadata of
```

Comment:
It's not quite that simple, this documentation code is executable. Hence, to load a different metadata and the corresponding model, we would need to train one first.